registry.json

```diff
@@ -2337,7 +2337,7 @@
     "id": "aurora",
     "name": "Aurora",
     "coinId": 1323161554,
-    "symbol": "AURORAETH",
+    "symbol": "AETH",
```

Comment:
Arbitrum and Optimism are also using just `ETH` on https://chainlist.org/ now, we need app to distinguish same name tokens by image

Reply:
but when it's the same ETH, what is the reason to have different names? to highlight the network it's on only, right?